### ISSUE TITLE
Some remaining questions with fine-pruning-celeba.py

### ISSUE THREAD

**Issue by HazardFY** [VinAIResearch/Warping-based_Backdoor_Attack-release#6]
Thanks for your answers. For the first question before, I find that I didn't download the latest version of the code. But I still have some questions about [https://github.com/VinAIResearch/Warping-based_Backdoor_Attack-release/tree/main/defenses/fine_pruning/fine-pruning-celeba.py](url)

1. [https://github.com/VinAIResearch/Warping-based_Backdoor_Attack-release/blob/main/defenses/fine_pruning/fine-pruning-celeba.py#L83](url) **opt.input_width**  is not assigned.
2. When I apply the fine_pruning method in ResNet50, I find that I have to redefine the last **bn** layer with code  `nn.BatchNorm2d(pruning_mask.shape[0] - num_pruned)` and then load the  **bn**'s params data in the way of   https://github.com/VinAIResearch/Warping-based_Backdoor_Attack-release/blob/main/defenses/fine_pruning/fine-pruning-celeba.py#L150](url). Otherwise, the output of redefined last **conv** layer doesn't match the dimension of input of the last **bn** layer. Finally before using  **net_pruned** in the eval function, I used  `net_pruned.eval()` to fix params of the redefined last **bn** layer.(The Resnet50 which I used is `torchvision.models.resnet50()`, so the dimension of the concrete layer may be different, but I think redefining **bn** layer perhaps is also needed in your code)
